tests/unit/AiClientTest.php

```diff
@@ -30,7 +30,8 @@ protected function setUp(): void
 
     protected function tearDown(): void
     {
-        // Tests use dependency injection - registry instances passed directly to methods
+        // Clean up static event dispatcher after each test
+        AiClient::setEventDispatcher(null);
```

Comment:
Why does this need to be cleaned up here? I don't see anything in this test class where it's set?